dts/arm/st/u5/stm32u595.dtsi

```diff
@@ -115,7 +115,9 @@
 
 	otghs_phy: otghs_phy {
 		/* Clock source defined by USBPHYC_SEL in */
-		compatible = "usb-nop-xceiv";
+		compatible = "st,stm32-otghs-phy";
+		/* Default to assuming the HSE clock is 16MHz */
+		clock-cfg = <STM32U5_OTGHS_PHY_CLKSEL_16MHZ>;
```

Comment:
This sounds board-specific and, if so, should not be specified at SoC level.

Reply:
Would you like me to just delete it? HSE is board specific, but the driver made the assumption that it was 16MHz.

Reply:
Deleted the section!